src/gui/windows/download_controller_mappings.zig

```diff
@@ -0,0 +1,42 @@
+const std = @import("std");
+
+const main = @import("root");
+const files = main.files;
+const settings = main.settings;
+const Vec2f = main.vec.Vec2f;
+
+const gui = @import("../gui.zig");
+const GuiComponent = gui.GuiComponent;
+const GuiWindow = gui.GuiWindow;
+const Button = @import("../components/Button.zig");
+const CheckBox = @import("../components/CheckBox.zig");
+const Label = @import("../components/Label.zig");
+const VerticalList = @import("../components/VerticalList.zig");
+const HorizontalList = @import("../components/HorizontalList.zig");
+
+pub var window = GuiWindow {
+	.contentSize = Vec2f{256, 256},
+	.closeable = false
+};
+
+const padding: f32 = 8;
+pub fn update() void {
+	if (!main.Window.gamepad.controllerMappingsDownloading()) {
```

Comment:
This is a double negatation (one here, the other inside the function) and I find it a bit confusing. Better change the function name to `controllerMappingsDownloaded` and remove the two negations.

Reply:
Changed it to wereControllerMappingsDownloaded to prevent a name collision with the value it references.